### PR TITLE
update delius config for contact log notifications

### PR DESCRIPTION
**Pull request by tomsmyers**
## What does this pull request do?

update delius config for contact log notifications

## What is the intent behind these changes?

get the 'referral sent' notification showing up in delius.

### CODE REVIEW

**Review by mxco86 — CHANGES_REQUESTED** [review]

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ env:
   HMPPSAUTH_BASEURL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   INTERVENTIONSUI_BASEURL: "https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk"
   COMMUNITYAPI_BASEURL: "https://community-api-secure.test.delius.probation.hmpps.dsd.io"
-  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_PROBATIONAREACODE: "A00"
-  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_REFERRALTYPE: "C385"
-  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_STAFFCODE: "ASPUATU"
-  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE: "ASPUAT"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_PROBATIONAREACODE: "CRS"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_REFERRALTYPE: "CRS01"
+  COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_STAFFCODE: "CRSUAT"
```
[the remark this answers is not in the document]
Comment (tomsmyers) [reply helm_deploy/values-dev.yaml:24]: should the team code match?

[the remark this answers is not in the document]
Comment (mxco86) [reply helm_deploy/values-dev.yaml:24]: No - team code is correct as it is

**Review by mxco86 — DISMISSED** [review]

File: helm_deploy/values-dev.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomsmyers) [reply helm_deploy/values-dev.yaml:24]: sorted, thanks.

**Review by mxco86 — APPROVED** [review]
👍

**Review by sldblog — APPROVED** [review]
👍 LGTM; could we leave a note on why the codes changed, please? (For me, it's important to know what external change has to be reflected in the future)

**Comment by tomsmyers** [comment]
> 👍 LGTM; could we leave a note on why the codes changed, please? (For me, it's important to know what external change has to be reflected in the future)

paging @mxco86  for this

**Comment by mxco86** [comment]
> 👍 LGTM; could we leave a note on why the codes changed, please? (For me, it's important to know what external change has to be reflected in the future)

The codes have now been confirmed on the Delius side as-per this:https://github.com/ministryofjustice/hmpps-interventions-service/pull/115

**Pull request merged**